sailor/sap_iot/fetch.py

```diff
@@ -236,6 +247,10 @@ def get_indicator_data(start_date: Union[str, pd.Timestamp, datetime.timestamp,
         if request_ids:
             time.sleep(5)
             print('.', end='')
+
+        if timeout is not None and timeout < (time.time() - start_time):
+            LOG.debug('Timeout of %d seconds was reached for fetching indicator data.', timeout)
+            break
```

Comment:
currently breaks out of the loop silently. I think this warrants at least a `warning`-level message, but possibly even an error. @cloudyday?

Reply:
Yes. If no error is raised, having a timeout won't not add any value.

The user expects the call to be successful within `timeout`, if the call takes longer it is an **error**. And the user can then choose how to handle that error.

Reply:
Log message changed to a RuntimeError.